# We can use the try/except statements to handle the error